senator/game.py

```diff
@@ -38,37 +47,47 @@ def sample_joint_action(self) -> np.ndarray:
     def _get_instaneous_regret(self, player: int, strategy: np.ndarray) -> np.ndarray:
         opponents = [i for i in range(self._num_players) if i != player]
         utility_matrix = self.utility_matrix[player]
-        for opponent in reversed(opponents):  # We start from the last opponent because `np.tensordot` collapses axes
-            utility_matrix = np.tensordot(utility_matrix, strategy[opponent], axes=([opponent], [0]))  # TODO: check if this is correct
+        for opponent in reversed(
+            opponents
+        ):  # We start from the last opponent because `np.tensordot` collapses axes
+            utility_matrix = np.tensordot(
+                utility_matrix, strategy[opponent], axes=([opponent], [0])
+            )  # TODO: check if this is correct
```

Comment:
Jestli jsi to checknul, můžeme tohle smazat :)

Reply:
S Filipem jsme to nějak řešili hned ten první den, co jsi nám to posílal, myslím, že ok.

Reply:
Jo tohle TODO smazat